proof/invariant-abstract/Tcb_AI.thy

```diff
@@ -371,6 +373,9 @@ lemma zombies_final_helperE:
   apply (fastforce simp: cte_wp_at_caps_of_state)
   done
 
+context Tcb_AI_1
+begin
```

Comment:
For these ones (that don't define a locale), I'd prefer the `begin` to be on the same line, or at least that's how I've been doing it in arch-split of Refine...